lib/model.d.ts

```diff
@@ -325,7 +325,7 @@ export interface IncludeOptions {
   /**
    * Load further nested related models
    */
-  include?: Includeable[];
+  include?: Includeable[] | IncludeAll[];
```

Comment:
Adding the extra typing to `Includeable` directly appears to allow for this to compile though:
```TypeScript
MyModel.findAll({
  include: [{
    all: true,
    include: [{
      all: true
    }]
  }]
})
```
Since that's obviously undesired behavior, we might still have to resort to using `export type Includeables = Includeable[] | IncludeAll[];` - that seems to be the only option I can get my `tsc` version to accept.